python/featomic/tests/clebsch_gordan/cartesian_spherical.py

```diff
@@ -1,9 +1,11 @@
 import numpy as np
 import pytest
-from metatensor import Labels, TensorBlock, TensorMap
+from metatensor import Labels, TensorBlock, TensorMap, operations
```

Comment:
Please don't import operations directly, the functions should be used as `metatensor.drop_blocks`/`mts.drop_blocks`

```suggestion
from metatensor import Labels, TensorBlock, TensorMap
import metatensor as mts
```